style/js/shield_defs.js

```diff
@@ -718,6 +718,34 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Andrews:Andrews:Loop"] = banneredShield(roundedRectShield("white", "#003f87", "#003f87", 1, 1, null), ["LOOP"]);
+  shields["US:TX:Loop"] = banneredShield(shields["US:TX"], ["LOOP"]);
+  shields["US:TX:Spur"] = banneredShield(shields["US:TX"], ["SPUR"]);
+  shields["US:TX:Business"] = banneredShield(shields["US:TX"], ["BUS"]);
+  shields["US:TX:Park"] = banneredShield(shields["US:TX"], ["PARK"]);
+  shields["US:TX:Beltway"] = banneredShield(shields["US:TX"], ["BELT"]);
+  shields["US:TX:FM"] = banneredShield(shields["US:TX"], ["F.M."]);
+  shields["US:TX:FM:Business"] = banneredShield(shields["US:TX:FM"], ["BUS, F.M."]);
+  shields["US:TX:PA"] = banneredShield(shields["US:TX"], ["P.A."]);
+  shields["US:TX:RM"] = banneredShield(shields["US:TX"], ["R.M."]);
+  shields["US:TX:Recreational"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R"]
+  );
+  shields["US:TX:NASA"] = banneredShield(shields["US:TX"], ["NASA"]);
+
+  shields["US:TX:Toll"] = banneredShield(
+    roundedRectShield("#003f87", "white", "white", 1, 1, null),
+    ["TOLL"]
+  );
+  shields["US:TX:BCTRA"] = shields["US:TX:Toll"];
+  shields["US:TX:CTRMA"] = shields["US:TX:Toll"];
+  shields["US:TX:NTTA"] = shields["US:TX:Toll"];
+  shields["US:TX:Express:Toll"] = banneredShield(shields["US:TX:Toll"], ["TOLL", "EXPR"]);
+  shields["US:TX:Loop:Toll"] = banneredShield(shields["US:TX:Toll"], ["TOLL", "LOOP"]);
+  shields["US:TX:Loop:Express:Toll"] = banneredShield(shields["US:TX:Toll"], ["TOLL", "EXPR", "LOOP"]);
```

Comment:
"TOLL" is usually below the shield. Obviously we don't support modifiers below the shield, and "LOOP" should probably remain last in order to indicate that these are really State Loops with some modifiers. But IMO "EXPRESS" should go before "TOLL".

Is `US:I:Express:Toll` still missing?

Reply:
The `network` values for those relations were tagged by me, so don't read too much into the ordering. I think it's more important to match the order displayed on the signs. If that gives the shield and modifiers an hourglass figure, so be it.